events/code_review_events/workflow.py

```diff
@@ -257,6 +262,16 @@ def __init__(self, cache_root):
         )
         self.monitoring.register(self.bus)
 
+        # Create pulse listener for unit test failures
+        self.pulse = PulseListener(
```

Comment:
Please rename to `pulse_unit_tests` or anything with a suffix, as i'll need `pulse_autoland` pretty soon !

Reply:
You might be able to use the same listener, depending on how you want to implement that.